src/Support/AliasArguments/AliasArguments.php

```diff
@@ -13,9 +13,6 @@
 
 class AliasArguments
 {
-    private $typedArgs;
-    private $arguments;
-
```

Comment:
Removed them right in this PR as I noticed they were unused after the code was refactored